analysis/monkmistweaver/src/integrationTests/example.test.ts.snap

```diff
@@ -6035,7 +6035,7 @@ exports[`Mistweaver Monk integration test: example log MistweaverHealingEfficien
           <td
             className="text-right"
           >
-            2080000
+            2.08m
```

Comment:
this entire file diff seems troublesome

Reply:
The current version of the mana efficiency module only includes spells that have mana costs defined on the spell. I changed it so that it grabs the mana costs from the casts. There is more data being shown for mana efficiency now. 

I also fixed an issue with some infinite values breaking the chart, and added some number formatting to make it cleaner.

Reply:
That's not the only issue in it though, such as further below has a NaN and Infinity whereas it had numbers before:

![image](https://user-images.githubusercontent.com/4565223/126606030-51477a79-b088-4762-aa13-3233621e6313.png)